sdk/monitor/azure-monitor-opentelemetry/pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.azure-sdk-build]
+type_check_samples = false
+verifytypes = false
+pyright = false
+mypy = false
+pylint = false
+bandit = False
```

Comment:
your `bandit` config doesn't have a `false` yml value. This is probably evaluating to true, which is why `bandit` is still running against your PR.